### ISSUE TITLE
dont work neural AI network

### ISSUE THREAD

**Issue by brakdag** [chazomaticus/netttt#1]
I try play with neural network AI, but, don't respond.
why?

**Comment by JonErikDSuero** [comment]
Try Firefox. It seems that Chrome is blocking the workers.

**Comment by chazomaticus** [comment]
Yeah, I probably only ever tested in Firefox.  Assuming that's what @brakdag was running into, I'll edit this issue to be a little more descriptive.  Please correct me if I'm wrong.
